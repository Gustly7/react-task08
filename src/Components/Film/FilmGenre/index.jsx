import React, { Component } from 'react'

class FilmGenre extends Component {
  render () {
    return (
      <div className='FilmGenre'>
        {this.props.filmGenre}
      </div>
    )
  }
}

export default FilmGenre
