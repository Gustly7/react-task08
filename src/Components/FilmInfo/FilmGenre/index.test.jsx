import React, { Component } from 'react'
import FilmGenre from './index'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

describe('FilmGenre component test suite', () => {
  test('FilmGenre snapshot renders correctly', () => {
    const tree = renderer.create(
      <FilmGenre />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('FilmGenre div should have "FilmGenre" class', () => {
    const wrapper = shallow(
      <FilmGenre />
    )
    const elem = wrapper.find('div.FilmGenre')
    expect(elem.length).toBe(1)
  })
})
