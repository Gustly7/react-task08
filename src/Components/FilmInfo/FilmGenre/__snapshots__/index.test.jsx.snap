// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmGenre component test suite FilmGenre snapshot renders correctly 1`] = `
<div
  className="FilmGenre"
/>
`;
